forkpi/spoonpi/spoonpi/forkpi_db.py

```diff
@@ -33,6 +35,20 @@ def authenticate(self, *args, **kwargs):
              This is an empty list if is_authorized is False
         """
         keywords = kwargs.keys()
+        GLOBAL_PIN = "global_pin"
+        PIN = "pin"
+        RFID_UUID = "rfid_uid"
+        FINGERPRINT_MATCHES = "fingerprint_matches"
```

Comment:
Can we move this constants in the global space? We can also create and enum in the models for these.